### PR TITLE
Timestamp in embeds and raise error in commands

### PR DESCRIPTION
**Pull request by Amey-wale**
Added timestamps in embeds and added raise error in errors of commands link clear messages, ban, kick so that any unknown error can be seen in the terminal !.

its_charmander

### CODE REVIEW

File: Rule.py [hunk 1]
```diff
@@ -73,7 +74,8 @@ async def rule_begins(self, ctx):
 
 		msg = await chnl.send(embed = discord.Embed(
                 description = rule_message3,
-                colour = color)
+                colour = color,
+				timestamp=ctx.message.created_at)
```
Comment (ZeusAbhijeet) [inline Rule.py:78]: timestamps not required for this one because it won't suit it.

File: Util.py [hunk 2]
```diff
@@ -82,7 +82,8 @@ async def command_log(client, ctx, cmd_name):
 	embed = discord.Embed(
 		title = "SleepBot Command Logs",
 		description = ("Command: {}\nMessage Content: {}".format(cmd_name, ctx.message.content)),
-		colour = random.randint(0, 0xffffff)
+		colour = random.randint(0, 0xffffff),
+		timestamp=ctx.message.created_at
```
Comment (ZeusAbhijeet) [inline Util.py:86]: Logs already have the time the command was run so not required

File: CodeHelp.py [hunk 3]
```diff
@@ -45,7 +48,8 @@ async def ask(self,ctx, result_limit: typing.Optional[int] = 3, *, term: str=Non
 
                 answerEmbed=discord.Embed(
                     title='Answers',
-                    colour=embedColour
+                    colour=embedColour,
+                    timestamp=ctx.message.created_at
```
Comment (ZeusAbhijeet) [inline CodeHelp.py:52]: Timestamps won't work out for this embed because page number will overwrite it

**Review by ZeusAbhijeet — CHANGES_REQUESTED** [review]
Remove the timestamps where I have added comments in `CodeHelp.py`, `Rule.py` and `Util.py` because they either don't suit, won't show up or are already there in some other form

File: CodeHelp.py (hunk 3 above)
Comment (Amey-wale) [reply CodeHelp.py:52]: I have edited the files , I had committed them too, you can check !

**Comment by Amey-wale** [comment]
Updated the files

**Pull request merged**